package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
package.json doesnt have react lib even though u use react

Reply:
This file is created automatically so I don't really know...